sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -1193,7 +1193,7 @@ def test_sincos_rewrite_sqrt():
     for p in [1, 3, 5, 17]:
         for t in [1, 8]:
             n = t*p
-            for i in range(1, (n + 1)//2 + 1):
+            for i in range(1, min((n + 1)//2 + 1, 10)):
```

Comment:
I understand the specific values, it's just confusing to me.  I suspect it would be confusing to someone else trying to figure out what this test is doing and why it was written this way.  If it stays as a nested loop, I would suggest adding a comment explaining that you're just capping it off for speed reasons.